### ISSUE TITLE
Porting twisted.application.service to python3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6916]
|<img alt="realcr's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| realcr reported|
|-|-|
|Trac ID|trac#6916|
|Type|enhancement|
|Created|2014-01-18 10:51:40Z|

service.py will support both python2.7 and python3.3 syntax.

Uploaded patch contains:
- Addition of relevant from __future__ statement.
- Changes of implements to implementer, regarding zope.interface.

Attachments:

* [application_service_py3.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/7dc/7dc2dc37305503eed0b4036d8536018fa3465eaa/54fedd4b540eb7301b567f96a9dc69dacffd380b.patch) (2020 bytes) - added by realcr on 2014-01-18 10:52:05Z - Minor syntax fixes.

<details><summary>Searchable metadata</summary>

```
trac-id__6916 6916
type__enhancement enhancement
reporter__realcr realcr
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1390042300000000 1390042300000000
changetime__1435821340446499 1435821340446499
version__None None
owner__realcr realcr

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to realcr|
|-|-|

Comments for this are basically the same as for [#6910](https://github.com/twisted/twisted/issues/6910).  Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl set status to `closed`|
|-|-|

Ported in the merged [#7817](https://github.com/twisted/twisted/issues/7817).
